src/System.Security.Cryptography.Algorithms/tests/RSATests.cs

```diff
@@ -192,6 +193,28 @@ public void VerifyDataStream_UsesHashDataAndVerifyHash()
             Assert.True(rsa.VerifyData(new MemoryStream(new byte[] { 42 }), new byte[1] { 24 }, HashAlgorithmName.SHA256, RSASignaturePadding.Pkcs1));
         }
 
+        [Fact]
+        public void RSAEncryptionPadding_Equality()
+        {
```

Comment:
Why are we adding this test in this PR?

Reply:
RSAEncryptionPadding equality is currently untested and is changed in the PR.